pkg/manifest/manifest.go

```diff
@@ -187,32 +188,28 @@ func resolveRelativeLinks(node *Node, _ *Node, manifest *Node, r registry.Interf
 	return nil
 }
 
-func checkFileTypeFormats(node *Node, _ *Node, manifest *Node, r registry.Interface, ContentFileFormats []string) error {
+func checkFileTypeFormats(node *Node, _ *Node, manifest *Node, r registry.Interface, contentFileFormats []string) error {
 	if node.Type != "file" {
 		return nil
 	}
-	file := node.FileType.File
-	isValidTypeFormat := false
-	for _, fileFormat := range ContentFileFormats {
-		if strings.HasSuffix(file, fileFormat) {
-			isValidTypeFormat = true
+	files := append(node.FileType.MultiSource, node.FileType.Source, node.FileType.File)
+	for _, file := range files {
+		if !slices.ContainsFunc(contentFileFormats, func(fileFormat string) bool { return strings.HasSuffix(file, fileFormat) || file == "" }) {
```

Comment:
Can we comment why we do `|| file == "" `
Something like: `// skip empty files`